composer.json

```diff
@@ -29,8 +29,9 @@
     "oat-sa/generis": ">=14.2.0",
     "oat-sa/tao-core": ">=48.0.0",
     "oat-sa/extension-tao-delivery": ">=14.10.1",
-    "oat-sa/extension-tao-outcomeui": ">=10.4.0"
-  },
+    "oat-sa/extension-tao-outcomeui": ">=10.4.0",
+    "oat-sa/lib-tao-elasticsearch": "dev-develop"
```

Comment:
We need to add the proper tag 1.2.0 https://github.com/oat-sa/lib-tao-elasticsearch/commit/fec4433d0e5409a331eb7d306f67d68cc05a34db